java/piranha/src/test/java/com/uber/piranha/XPFlagCleanerTest.java

```diff
@@ -2120,4 +2120,136 @@ public void exceptionWhenArgumentIndexNotSpecifiedAndNotArgumentIndexOptional()
     XPFlagCleaner flagCleaner = new XPFlagCleaner();
     flagCleaner.init(b.build());
   }
+
+  @Test
+  public void testEmptyFlagRemovesStaticImports() throws IOException {
```

Comment:
nit: test needs to be either renamed to `testEmptyFlagAndStaticImports` or `testEmptyFlagDoesNotRemoveStaticImports`. The latter is descriptive but more verbose. If you go for the former, please add a comment before the test.